src/tensors/cpu/wasm_intgemm_interface.h

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+/* Main interface for integer matrix multiplication (C = A * B) for wasm.
+ *
+ * A is typically activations whose rows should be a multiple of 1 (i.e. no restriction) and
+ * columns should be a multiple of 64.
+ * 
+ * B is typically fixed model parameters whose rows should be a multiple of 64 and columns
+ * should be a multiple of 8.
```

Comment:
A question for documentation purpose. Would A and B matrices also be in row-major format?

Reply:
We assume everything is in a Row Major format.

Reply:
Great. I will document it then 👍